engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -275,4 +278,21 @@ public static synchronized void changeLoggersLevel(@Nonnull Map<String, String>
     }
     return ImmutableMap.copyOf(matchedClasses);
   }
+
+  public static String getLogFileName()
+  {
+    ApexRollingFileAppender apexAppender = (ApexRollingFileAppender)LogManager.getRootLogger().getAppender(
```

Comment:
It shouldn't as I am looking for appender with name "APEXRFA". Still I would add a check for safety.